Resources/Textures/Structures/Machines/tabletop_computers.rsi/meta.json

```
@@ -0,0 +1,27 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/bd6873fd4dd6a61d7e46f1d75cd4d90f64c40894, computers modified by JustUser.",
```

Uh, no. These are not your sprites.
https://github.com/new-frontiers-14/frontier-station-14/blob/master/Resources/Textures/_NF/Structures/Machines/computer_tabletop.rsi/meta.json#L4

Please do not misattribute CC-BY-SA work.
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/bd6873fd4dd6a61d7e46f1d75cd4d90f64c40894. | Resprited by erhardsteinhauer (discord)",
```